README.md

```diff
@@ -108,14 +106,46 @@ The bundle does add a `certManager` block, that allow to change the version (cur
 
 # Specify the bundle to use
 bundles:
-- targets:
-  - run://quay.io/kairos/community-bundles:cert-manager_latest
+  - targets:
+      - run://quay.io/kairos/community-bundles:cert-manager_latest
 
 # Specify cert-manager settings
 certManager:
   version: v1.11.0
 ```
 
+### Flux
+
+This installs Flux and supports bootstrapping the cluster with [FluxCD](https://fluxcd.io/flux/cmd/flux_bootstrap/). It has logic so only one node will do the bootstrapping,
+by creating a ConfigMap in the `default` namespace named `flux-bootstrap`.
+By default, it will time out after trying for 30 minutes.
+
+```yaml
+#cloud-config
+
+bundles:
+  - targets:
+      # Installs to /usr/local/lib/extensions/flux/
+      - container://quay.io/kairos/community-bundles:flux_latest
+
+stages:
+  boot:
+    - name: "Bootstrap with Flux"
+      commands:
+        - bash /usr/local/lib/extensions/flux/bootstrap.sh &
```

Comment:
If it ran in a run.sh script, it blocked booting in my testing because the script blocks until it either bootstraps or confirms that instance won't be doing the bootstrapping.

The root problem is flux needs to be bootstrapped after the cluster has started and I couldn't figure out where to define running it immediately after the cluster is bootstrapped. I'd love for it to match the `run://` convention with no extra steps.